assets/css/styles.css

```diff
@@ -168,6 +168,8 @@ hr {
 .pv1 {
   display: flex;
   flex-flow: row nowrap;
+  justify-content: center;
+  align-items: center;
   gap: 1rem;
 }
```

Comment:
I looked at [tachyons docs](https://tachyons.io/docs/table-of-styles/) recently and realized that this is a bad idea.

We'll fix it soon in a future update.